src/category/functor.lean

```diff
@@ -42,7 +42,8 @@ def id.mk {α : Sort u} : α → id α := id
 
 namespace functor
 
-@[nolint] def const (α : Type*) (β : Type*) := α
+@[nolint unused_arguments doc_blame]
```

Comment:
`@[nolint doc_blame]` is saying this def is permanently allowed to not have a doc string. Could you either add a doc string, or add it to `nolints.txt` (temporary permission to not have a doc string)? Same for `map` below. The examples in `ordinal` are a little different since morally they're lemmas, not defs.

I'd suggest a doc string but honestly I'm not sure what this is for.

Reply:
Good point!  I've added a docstring for `const`, but the rest of the file is a bit of a mystery to me as well.  I've got no idea what `add_const` is for.

Reply:
Maybe @cipher1024 could help here.